autoload/fish.vim

```diff
@@ -26,6 +26,8 @@ function! fish#Format()
         let l:command = v:lnum.','.(v:lnum+v:count-1).'!fish_indent'
         echo l:command
         execute l:command
+        " Fix indentation and replace tabs with spaces if necessary.
+        normal! '[=']
```

Comment:
Should probably use `:retab` for this. Why do you feel you need to fix the indentation of the output from the program that is made for fixing indentation? Does it get something wrong?